builtin/aws/ecr/registry.go

```diff
@@ -63,18 +66,48 @@ func (r *Registry) Push(
 	}
 	svc := ecr.New(sess)
 
+	repoName := r.config.Repository
+
+	if repoName == "" {
+		log.Info("infering ECR repo name from app name")
+		repoName = "waypoint-" + src.App
```

Comment:
Since we're creating an ECR repo for the user here, I think it would be great to give them a bit of extra feedback on this with the generated repo name in the step output rather than just at the info level so they know a resource has been auto-magically created for them :smile:

Reply:
Totally. I've updated it to use a StepGroup and give a bunch more feedback about these sorts of things.